### PR TITLE
Adds isMatch to lodash definitions

### PR DESCRIPTION
**Pull request by rodeoclown**

### CODE REVIEW

File: lodash/lodash.d.ts [hunk 1]
```diff
@@ -6412,6 +6412,7 @@ declare module _ {
         endsWith(str?: string, target?: string, position?: number): boolean;
         escape(str?: string): string;
         escapeRegExp(str?: string): string;
+		isMatch(str?: string, target?: string): boolean;
```
Comment (vvakame) [inline lodash/lodash.d.ts]: https://lodash.com/docs#isMatch
this function? `isMatch(str?: any, target?: any): boolean;` ?

Comment (rodeoclown) [inline lodash/lodash.d.ts]: Yes, I left out the `customizer` and the `thisArg` because this is only being used for string comparisons.

Comment (vvakame) [inline lodash/lodash.d.ts]: I think your definition can't compile `_.isMatch(object, { 'age': 36 });`. isn't it?

Comment (rodeoclown) [inline lodash/lodash.d.ts]: Good point - I put it in purely thinking about strings.

**Pull request closed without merging**